ticketreminder/__init__.py

```diff
@@ -1,4 +1,4 @@
 from api import *
 import pkg_resources
 
-pkg_resources.require('Trac >= 1.0')
+pkg_resources.require('Trac >= 1.3')
```

Comment:
The changes you made should be compatible with 1.2 as well. Could we set this to 1.2?

Reply:
If you can test it, sure